django_cloud_tasks/connection.py

```diff
@@ -12,14 +13,16 @@ class DiscoveryCache:
 
     See https://github.com/googleapis/google-api-python-client/issues/325#issuecomment-419387788
     """
+
```

Comment:
this whole class (DiscoverCache) can go away

Reply:
fixed